regulations/check.py

```diff
@@ -0,0 +1,122 @@
+import sys
+
+
+class translationChecker():
+
+  def __init__(self):
+    sys.stdout.flush()
+    self.regulations_set = self.readFile('wca-regulations/wca-regulations.md', False)
+    self.guidelines_set = self.readFile('wca-regulations/wca-guidelines.md', True)
+
+  # If isGuidelines is False, check that every regulation is indented
+  # according to its number, that every lind number doesn't end with '+'.
+  #
+  # If isGuidelines if True, check that every regulation has no
+  # indentation, that every line number ends with '+', that every
+  # line has a label, and there are six different labels.
+  #
+  # Returns set of regulations numbers.
+  def readFile(self, filename, isGuidelines):
+    numbers_set = set()
+    found_table_of_contents = False
+    labels_set = set()
+
+    with open(filename, 'r') as input_file:
+      for line_number, line in enumerate(input_file):
+        line_number += 1
+        line = line.rstrip('\n')
+
+        # Skip everything before <table-of-contents>.
+        if not found_table_of_contents:
+          if line == '<table-of-contents>':
+            found_table_of_contents = True
+          continue
+
+        # Ignore empty lines or lines that start with '<' or '#'.
+        if line == '' or line[0] == '<' or line[0] == '#':
+          continue
+
+        # Try to extract regulation number. If there is no number, skip.
+        ind1, ind2 = line.find('- '), line.find(')')
+        if ind1 == -1 or ind2 == -1:
+          continue
+        line_id = line[ind1 + 2 : ind2]
```

Comment:
nit: This 2 is a magic number. If I'm reading this correctly, I think this would be more self-documenting if you wrote it as:

```python
line_id = line[ind1 + len('- ') : ind2]
```

Although a regular expression with a group would probably be cleanest here.